src/lore_api_client.rs

```diff
@@ -36,17 +49,17 @@ impl PatchFeedRequest for BlockingLoreAPIClient {
 
         match reqwest::blocking::get(feed_request) {
             Ok(response) => feed_response = response,
-            Err(error) =>  return Err(FailedFeedRequest::UnknownError(error)),
+            Err(error) =>  bail!(FailedFeedRequest::UnknownError(error)),
```

Comment:
This may get a little complex:
 
The point of this PR is to use color_eyre's Result since we can cast any Report type (how color_eyre calls valid error types) into the Err variant.

This allows the `.text().unwrap()` at line 60 to become a `.text()?`, so instead of panicking we propagate the error when calling `.text()`.

So this function may return 2 different kinds of errors: a FailedFeedRequest  or a `reqwest::error::Error`. Those two errors are propagated as a `Report`, but the caller may use a `downcast` to recorver from this error